itests/hive-unit/pom.xml

```diff
@@ -489,6 +489,10 @@
       <version>${htmlunit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
```

Comment:
Either mark the dependency as optional or put it under test scope. Optional would keep things more uniform with the rest of the project but pick whatever you prefer.